modal/runner/containers/vllm_unified.py

```diff
@@ -31,14 +37,15 @@ def _make_container(
     else:
         raise ValueError(f"Unknown GPU type: {gpu}")
 
+    # Avoid wasting resources & money in dev
+    if keep_warm and is_env_dev():
+        keep_warm = None
+
     class _VllmContainer(VllmEngine):
-        def __init__(
-            self,
-            model_path: Path,
-            max_model_len: Optional[int] = None,
-        ):
```

Comment:
opted to simplify some more, avoiding any potential bugs in modals "parametrized class" keep_warm

we could have done it this style earlier too, but we hadnt reorganized to one-model-one-container yet